src/libs/ajax/WorkspaceDataService.ts

```diff
@@ -53,6 +53,21 @@ export interface WDSJob {
   updated: string;
 }
 
+// The source of truth of valid capabilities can be found in: https://github.com/DataBiosphere/terra-workspace-data-service/blob/main/service/src/main/resources/capabilities.json
+type KnownCapability =
+  | 'capabilities'
+  | 'dataimport.pfb'
+  | 'dataimport.tdrmanifest'
+  | 'edit.deleteAttribute'
+  | 'edit.renameAttribute';
+type UnknownCapability = string;
+export type Capability = KnownCapability | UnknownCapability;
+
+// Capabilities is just a kvp map of capability name to boolean. The value is true if the capability is enabled, false otherwise.
+export type Capabilities = {
+  [key in Capability]: boolean;
+};
+
```

Comment:
question for debate: is it worth it to list out the keys of the capabilities response here? Or could it just be a bag of string->boolean, no keys predefined? Will it be a maintenance headache to keep the list of keys up to date? Or maybe we only predefine keys here that the UI actually uses?

Reply:
Good call.  I think I like your last option which is to only predefine the keys here that are actually used by the UI.  Then the list will grow (and diffs will indicate) when new features were added, and (maybe?) shrink when features are perhaps able to be cleaned up after they are rolled out to 100% of WDS installs.

Reply:
Addressed in e4a21bfba419013aa9ef048760668967a02c9145

Reply:
Added a bit more clarification in 31a82168b62b56041ac2e5e6a3f4da37f1c0cf61